labcodes/confetti-storybook
# [CSYS-359] Migrating Icon to Hooks

Link to task:
[CSYS-355](https://labcodes.atlassian.net/browse/CSYS-355)

Staging app: https://csys-355-migrate-alert--confetti-storybook.netlify.app/

How to test:
After running your project locally, run the Icon.test.js to check if the Alert component is working as expected.

Description of your solution:

Removing previous mentions from Class Components and moving the Props outside the functional component scope.

Screenshots (when applicable): NA
